data/error_policies/parent_call.cas

```diff
@@ -0,0 +1,24 @@
+virtual resource bar {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+virtual resource zap {
+	fn write(domain source) {
+		allow(source, this, file, read);
```

Comment:
Doesn't overly matter to the point of the test, but the write() function should probably grant the write permission?